CHANGES.rst

```diff
@@ -27,6 +27,12 @@ Lemaitre <glemaitre>`, :user:`Adrin Jalali <adrinjalali>`, :user:`Olivier Grisel
 <ogrisel>` and others participated through many discussions in defining the
 requirements and the public API.
 
+- Added the `AdaptiveSquashingTransformer` that
```

Comment:
I think there is a formatting issue:

![screenshot_2025-04-15T13:32:29+02:00](https://github.com/user-attachments/assets/91926215-8ca8-4047-8c8f-90fcf7581a97)


the lines below the `-` need to be indented to still be part of that bullet point, and the last line is too indented which turns it into a block quote. yes, restructured text is an acquired taste. a couple of things that can help:

- https://docutils.sourceforge.io/docs/user/rst/quickref.html
- in the pr page in the checks at the bottom click on "check the rendered docs here!"
- or locally go in the `/doc/` folder and there `make html-noplot` to render the doc without actually running the examples. you may need to first `pip install ".[doc]"`

Reply:
also you can do 
```
:class:`AdaptiveSquashingTransformer`
```

that will turn it into a link to the documentation